extensions/wikia/Forum/ForumController.class.php

```diff
@@ -319,7 +358,7 @@ public function forumRelatedThreads() {
 
 			$rp = new WallRelatedPages();
 			$out = $rp->getMessageRelatetMessageIds( [ $title->getArticleId() ] );
-			$messages = [ ];
+			$messages = [];
 			$count = 0;
 			foreach ( $out as $key => $val ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$key" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)